test/utils.spec.js

```diff
@@ -0,0 +1,38 @@
+const utils = require('../packages/oc-template-handlebars/utils');
+
+describe('utils module', () => {
+  describe('when validating component', () => {
+    describe("when component's handlebars version is obsolete", () => {
+      const template = {
+        compiler: [6, '>= 2.0.0-beta.1'],
+        main: function() {
+          return 'Hello world!';
+        },
+        useData: !0
+      }; // eslint-disable-line
+      const result = utils.validator(template);
+
+      test('should error', () => {
```

Comment:
also in this case you can just snapshot the result as it is an object so that you don't have to copy/paste the message `The component can't be rendered because it was published with an older OC version`